tests/unit/pyramid_sanity/conftest.py

```diff
@@ -3,15 +3,15 @@
 
 import pytest
 from pyramid.registry import Registry
-from pyramid.request import Request
 from pyramid.response import Response
+from pyramid.testing import DummyRequest
 
 from pyramid_sanity._settings import SanitySettings
 
 
 @pytest.fixture
 def pyramid_request():
-    return create_autospec(Request, instance=True)
+    return DummyRequest()
```

Comment:
The tests were easier to write with a `DummyRequest` than with a mock. Also see this old comment about why mocking `Request` is a bad idea: https://github.com/hypothesis/h/pull/5274#discussion_r215934743